### PR TITLE
Change subscribe meta internal type from string to bytes

### PR DESCRIPTION
**Pull request by yilunzhang**
Protobuf bytes and string have the same encoding but different restrictions: string can now only contain utf-8 characters. However, the current chain data contains non-utf-8 string in meta generated using old pb version, which will cause new version fail to save certain blocks. Changing internal type to bytes will resolve the issue without breaking compatibility.

Signed-off-by: Yilun <zyl.skysniper@gmail.com>

### Type (put an `x` where ever applicable)
- [x] Bug fix: Link to the issue
- [ ] Feature (Non-breaking change)
- [ ] Feature (Breaking change)
- [ ] Documentation Improvement

### CODE REVIEW

**Comment by bufrr** [comment]
What is the point of this modification?

**Comment by yilunzhang** [comment]
The current version cannot save a few blocks containing non-uft8 subscribe metadata

File: pb/transaction.proto [hunk 1]
```diff
@@ -72,7 +72,7 @@ message Subscribe {
 	string topic       = 3;
 	uint32 bucket      = 4 [deprecated = true];
 	uint32 duration    = 5;
-	string meta        = 6;
+	bytes meta        = 6;
```
Comment (bufrr) [inline pb/transaction.proto:75]: looks like unformatted

Comment (yilunzhang) [reply pb/transaction.proto]: Fixed

**Review by bufrr — APPROVED** [review]

**Pull request merged**